pkg/commands/server/pluginshealth.go

```diff
@@ -44,11 +52,24 @@ var pluginsHealthCommand = cli.Command{
 // cmPluginsHealth is the action for the pluginsHealthCommand. It makes a "plugins" request
 // against the active Synse Server instance and returns plugins' health information.
 func cmdPluginsHealth(c *cli.Context) error {
-	plugins, err := client.Client.Plugins()
+	err := utils.RequiresArgsInRange(0, 3, c)
 	if err != nil {
 		return err
 	}
 
+	plugins, err := getPlugins(
+		c,
+		c.Args().Get(0),
+		c.Args().Get(1),
+		c.Args().Get(2),
+	)
```

Comment:
`c.Args()` is just a list of string, I believe (https://godoc.org/github.com/urfave/cli#Args), so you could just pass it in to the variadic like so:

```go
plugins, err := getPlugins(c, c.Args()...)
```

Reply:
This is much nicer. I didn't know that a list of string to the variadic function like so. I was checking using a counter earlier because the results I got from these separate argument could be an empty string.